### PR TITLE
Apply python best practices

### PR DESCRIPTION
**Pull request by Ullaakut**
Constant variables now have uppercase names
Functions now have snake_case names
Add python version in header
Remove useless imports
Remove unused variables
Ignore unused parameters
Add missing spaces
Remove trailing whitespaces
Shorten (some) lines over 100 columns (I stopped doing it after a while as almost all lines are extremely long)
Function local variables no longer override constant variables (might change program behavior, to check)

### CODE REVIEW

File: algo_eval.py [hunk 1]
```diff
@@ -28,64 +30,66 @@ def run_eval(t_id, algorithm, resize_factor, alpha, max_detectable_distance, mvt
...
+OBJ_RATIO = [0.41]
+SMOOTH_FILTER = [1]
+SMOOTH_FILT_SIZE = [7, 15]
+POST_FILTER = [1]
+POST_FILT_SIZE = [7, 15]
+MERGE_ALGO = [1]
+MERGE_MARGIN = [0.1]
+BG_ER_THRESH = [10, 15, 20, 25, 30, 35, 40]
+BG_DL_THRESH = [30]
+T_ID = 0
```
Comment (Ullaakut) [inline algo_eval.py:87]: Change back to lowercase

File: create_gt.py [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
 import collections
 import os.path
 
 # input parameters
-cam_id = 1
-tolerance = 100  # merge events being apart less or equal than this number of frames
-print_flag = False  # if False, avoid printing
-dataset_path = "/samples/CASA/videos/"  # dataset's root directory
-filelist = "/samples/CASA/gt/gt_cam" + str(cam_id).zfill(2) + ".txt"  # list of videos & event frames
+CAM_ID = 1
```
Comment (Ullaakut) [inline create_gt.py:7]: Change back to lowercase

File: create_gt.py [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
+TOLERANCE = 100  # merge events being apart less or equal than this number of frames
```
Comment (Ullaakut) [inline create_gt.py:8]: Change back to lowercase

File: create_gt.py [hunk 4]
```diff
@@ -1,51 +1,52 @@
...
+PRINT_FLAG = False  # if False, avoid printing
```
Comment (Ullaakut) [inline create_gt.py:9]: Change back to lowercase

File: create_gt.py [hunk 5]
```diff
@@ -1,51 +1,52 @@
...
 # Open file with list of frames with event(s)
-prefix = "frame_"
-suffix = ".png"
-prefix_len = len(prefix)
-suffix_len = len(suffix)
-videos = collections.defaultdict(list)
-paths = {}
+PREFIX = "frame_"
+SUFFIX = ".png"
+PREFIX_LEN = len(PREFIX)
```
Comment (Ullaakut) [inline create_gt.py:16]: Change back to lowercase

File: create_gt.py [hunk 6]
```diff
@@ -1,51 +1,52 @@
...
+SUFFIX_LEN = len(SUFFIX)
```
Comment (Ullaakut) [inline create_gt.py:17]: Change back to lowercase

**Comment by el97179** [comment]
Sorry brendy, but I had so many conflicts, it took me like 2h to sort them out and still some parts of the code don't work. As most of your work was "beautification", if it's ok I will reject this PR and I will apply most of your changes on another commit, on top of my current HEAD

**Pull request closed without merging**